fiasco/ions.py

```diff
@@ -940,7 +942,8 @@ def direct_ionization_cross_section(self, energy: u.erg) -> u.cm**2:
         :math:`U,f,\Sigma` are all stored in the CHIANTI database such that :math:`\sigma_I`
         can be computed for a given :math:`E`.
         """
-        if self.hydrogenic or self.helium_like:
+        if (self.hydrogenic and self.atomic_number >= 6) or
+            (self.helium_like and self.atomic_number >= 10):
```

Comment:
```suggestion
        if ((self.hydrogenic and self.atomic_number >= 6) or
            (self.helium_like and self.atomic_number >= 10)):
```

Reply:
Yeah, just noticed the tests failed.  Fixing it now...

Reply:
Too much C++ in my brain.